src/main/kotlin/org/jetbrains/intellij/platform/gradle/extensions/IntelliJPlatformDependenciesHelper.kt

```diff
@@ -793,6 +793,12 @@ class IntelliJPlatformDependenciesHelper(
         )
     }
 
+    // These two methods were created to make clearly visible that we want to use consistent version numbers in
+    // collectDependencies as well as in createIntelliJPlatformBundledModule & createIntelliJPlatformBundledPlugin
+    // When the version is retrieved like this, it is impossible to change it one place & forger to chang another.
+    private fun getVersionForBundledModule() = productInfo.get().buildNumber
```

Comment:
Given that it's indeed easy to make a mistake here, this was a good idea. However, given we have that method already, I hid the whole Ivy XML creation in such a method.